edb/tools/docs/js.py

```diff
@@ -0,0 +1,137 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2019-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from docutils import nodes as d_nodes
+from docutils.parsers.rst import directives
+
+from sphinx import addnodes as s_nodes
+from sphinx.domains import javascript as js
+from sphinx.locale import _
+from sphinx.util import docfields
+
+
+def merge_dicts(*dicts):
```

Comment:
You can just `merged = {**d1, **d2, **d3}`, or

```python
merged = dict(
  **d1,
  **d2,
  key1=val1,
  key2=val2
)
```

so no need for this.

Reply:
Right. I've updated the code and dropped the helper `merge_dicts`.